homeassistant/components/shelly/strings.json

```
@@ -3,7 +3,7 @@
     "flow_title": "{name}",
     "step": {
       "user": {
-        "description": "Before set up, battery-powered devices must be woken up by pressing the button on the device.",
+        "description": "Before set up, battery powered devices must be woken up, you can now wake the device up using a button on it.",
```

Battery-powered should have a hyphen in this case.

```suggestion
        "description": "Before set up, battery-powered devices must be woken up, you can now wake the device up using a button on it.",
```